src/makeRobotAccountant.js

```diff
@@ -6,7 +6,18 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counterOfCalls = 0;
+  const maxCall = 4;
```

Comment:
this variable is overkill=) you don't need it in such a small callback!)
```suggestion
```

Reply:
Ok, no problem ;)